beta/adobe-journey-optimizer/push-messaging-guide.md

```diff
@@ -0,0 +1,81 @@
+
+- Creating a datastream to configure the profile and experience event datasets against which the data flows into experience platform. 
+- Creating a client-side mobile property and adding extensions. Mobile SDK closely integrates with these extensions to provide a seamless data collection experience.
+- Registering the mobile app bundle identifier and app credentials that would help to uniquely identify and validate sanity of the app at the time of delivering push notification. 
+
+You will find detailed steps for these configurations [here](https://aep-sdks.gitbook.io/docs/beta/adobe-journey-optimizer).
+
+#### Real-time Customer Profile
+
+[Real-time Customer Profile](https://experienceleague.adobe.com/docs/experience-platform/profile/home.html) is the component in [Experience Platform](https://experienceleague.adobe.com/docs/experience-platform.html) that allows you to see a holistic view of each individual customer by combining data from multiple channels, including web, mobile, CRM, and third party. Profile allows you to consolidate your customer data into a unified view offering an actionable, timestamped account of every customer interaction. 
+
+Static data that identifies the user of the mobile app such as a push token, is stored against the user's profile as record data while the interactions the user does with push notifications are tracked as time-series events data. 
+
+#### Adobe Journey Optimizer
+
+Once your mobile app integrations with above mentioned components are in place and your customer profiles are available as Real-time customer profiles, you can take advantage of powerful audience segmentation capabilities in [Adobe Journey Optimizer](https://experienceleague.adobe.com/docs/journey-optimizer/using/ajo-home.html) to ensure the optimal experience for each person.
+
+
+
+### Push Data Flow Steps
+
+Refer to the data flow diagram to identify what each of these bullets represent :
+
+1. The customer develops a mobile app on Android or iOS that they would release to their users. In order to use the push capability provided by push providers i.e APNS by Apple and FCM by Google, the mobile app registers itself and enables the push capability.
+2. The push providers generate and send a push token to the mobile app. A Push Token is an identifier that senders use to target specific devices with a push notification.
+3. The mobile app is integrated with Adobe Mobile SDK which exposes various extensions and APIs. The Messaging extension exposes an API to register the push token into Adobe Experience Platform against customer's profile.
```

Comment:
"... integrated with Mobile SDK, which exposes...".

Also, does messaging need to be capitalized?